components/builder-api/src/server/resources/profile.rs

```diff
@@ -259,7 +259,7 @@ async fn set_license(req: HttpRequest,
                 };
 
             if !status.is_success() {
-                return HttpResponse::build(status).body(body);
+                return HttpResponse::build(status).body("Invalid license key.");
```

Comment:
Actually the way it was before is better. We should emit the error from the license server and not swallow it. If the user inputs a invalid key, from what I have seen, the api will return "License not found" which seems correct. However there may be some other circumstance where some other kind of error is thrown and we would want to have visibility into that.